src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -18955,5 +19045,160 @@ public void ExplicitCast_StaticType()
                 //     static object? G(object? y) => (C?)y;
                 Diagnostic(ErrorCode.ERR_ConvertToStaticClass, "(C?)y").WithArguments("C").WithLocation(4, 36));
         }
+
+        // PROTOTYPE(NullableReferenceTypes): Should only produce one CS8600
```

Comment:
I'm not sure about that. There is one warning for the cast (which leaves the null-state as "maybe null") and one for the assignment.
Trying to suppress one of the two warnings seems like it would compromise the design.